static/js/components/widgets/PeopleItem.js

```diff
@@ -25,7 +27,9 @@ export const PeopleItem: StatelessFunctionalComponent<Props> = ({
   <div className="person">
     <ProfileImage imageSize={PROFILE_IMAGE_MICRO} profile={profile} />
     <div className="description">
-      <span className="name">{profile.name}</span>
+      <Link to={profileURL(profile.username)} className="name navy">
```

Comment:
I just want to double-check that this will have the same underline-on-hover styling as other author links, e.g. in the post summary card, or the search results.